.github/workflows/ci.yml

```diff
@@ -12,24 +16,26 @@ jobs:
     steps:
 
     - name: Setup Go
-      uses: actions/setup-go@v1
+      uses: actions/setup-go@v3
       with:
-        go-version: 1.14
+        go-version: '1.14'
       id: go
+
     - name: Go module cache
-      uses: actions/cache@v2
+      uses: actions/cache@v3
       with:
         path: ~/go/pkg/mod
-        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        key: ${{ runner.os }}-go-${{ env.KREW_VERSION }}
```

Comment:
Why do we need to change this again?

Reply:
See https://github.com/kubernetes-sigs/krew-index/runs/6550071463?check_suite_focus=true#step:3:4 for example, you can see `key: Linux-go-`, i.e. `${{ hashFiles('**/go.sum') }}` does nothing.

Reply:
I think that'd be good to figure out why. We copied it from https://github.com/actions/cache/blob/main/examples.md so I expect it to work.

Reply:
Because this repo doesn't have go.mod/go.sum files. That example assumes you have a general Go module/package structure.

Reply:
D'uh I am not sure how that got past the code reviews initially. Thanks. You picked the right key here in that case. I hope we don't have the problem in Krew main repo but caching matters here more, definitely.